go.mod

```diff
@@ -0,0 +1,24 @@
+module aaha
+
+go 1.19
+
+require github.com/go-redis/redis/v8 v8.11.5
+
+require (
+	golang.org/x/mod v0.4.2 // indirect
+	golang.org/x/tools v0.1.1 // indirect
+	golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1 // indirect
+)
+
+require (
+	github.com/alicebob/gopher-json v0.0.0-20200520072559-a9ecdc9d1d3a // indirect
+	github.com/alicebob/miniredis v2.5.0+incompatible // indirect
+	github.com/cespare/xxhash/v2 v2.1.2 // indirect
+	github.com/dgryski/go-rendezvous v0.0.0-20200823014737-9f7001d12a5f // indirect
+	github.com/golang/mock v1.6.0
+	github.com/gomodule/redigo v1.8.9 // indirect
+	github.com/julienschmidt/httprouter v1.3.0 // indirect
+	github.com/sirupsen/logrus v1.9.0 // indirect
+	github.com/yuin/gopher-lua v0.0.0-20220504180219-658193537a64 // indirect
+	golang.org/x/sys v0.0.0-20220715151400-c0bba94af5f8 // indirect
+)
```

Comment:
folder `vendor` should not be commited to the repository

Reply:
As far as I understand go doesn't have a central artifact repo so if someone is removing his library from GitHub I would be able to build my code. That's why I added it here, but yeah is not a big deal to remove it.

Reply:
Yes, if someone deletes its repo from github - we're OOPS :) But this a very edge case.
You can store locally snapshot of vendor for emergency case, but in real world:
- deps are declared in `go.mod` (with versions specified)
- if someone pulls repo, he does `go mod download` - deps are downloaded (not in the repo folder, but outside the project) and that's it.

P.S. my chrome almost died when I opened PR diff, because of showing all those `vendor` files in diff view :)

Reply:
removed.